client/rest-high-level/src/main/java/org/elasticsearch/client/TasksRequestConverters.java

```diff
@@ -54,4 +55,12 @@ static Request listTasks(ListTasksRequest listTaskRequest) {
             .putParam("group_by", "none");
         return request;
     }
+
+    static Request getTask(GetTaskRequest getTaskRequest) {
+        Request request = new Request(HttpGet.METHOD_NAME, "_tasks/" + getTaskRequest.getNodeId() + ":" + getTaskRequest.getTaskId());
```

Comment:
can you use the EndpointBuilder here?

Reply:
Also added a method to EndpointBuilder for colon separated parts